Aquifer.xcodeproj/project.pbxproj

```diff
@@ -1006,7 +1003,6 @@
 				GCC_NO_COMMON_BLOCKS = YES;
 				INFOPLIST_FILE = Aquifer/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.4;
```

Comment:
This might be it.  On OS X this is fine, but iOS will flip a shit if it can't find the right sim build with this field.